c2corg_ui/templates/waypoint/edit.html

```diff
@@ -174,7 +174,7 @@ <h2 class="heading show-phone"><span translate>geolocation</span></h2>
               <label><span translate>longitude</span> *</label>
               <div class="input-group">
                 <input type="number" name="longitude" ng-model="waypoint.lonlat.longitude" ng-change="editCtrl.updateMap()" class="form-control" required required min="-90" max="90" />
-                <span class="input-group-addon">°</span>
+                <span class="input-group-addon">E°</span>
```

Comment:
Rather ``°E`` and ``°N``
https://en.wikipedia.org/wiki/Geographic_coordinate_system#/media/File:This_is_a_basic_photo_showing_that_latitude_lines_are_horizontal_and_longitude_lines_are_vertical-_2014-07-25_20-36.jpg